bin/rialto-parachain/runtime/src/lib.rs

```diff
@@ -777,6 +777,13 @@ impl_runtime_apis! {
 
 	#[cfg(feature = "runtime-benchmarks")]
 	impl frame_benchmarking::Benchmark<Block> for Runtime {
+		fn benchmark_metadata(_extra: bool) -> (
+			Vec<frame_benchmarking::BenchmarkList>,
+			Vec<frame_support::traits::StorageInfo>,
+		) {
+			todo!("TODO: fix or remove")
```

Comment:
We don't have any tests for benchmarking on CI, so this could break existing benchmarks (I know nothing about this method - so not sure). Can you, please, file an issue so that we (I) can test and fix it in the near future? Thanks!

Reply:
@svyatonik created https://github.com/paritytech/parity-bridges-common/issues/1588,
could you please also check and update it?

probably for rialto-parachain we dont need any benchmarks, but for bridges pallets used by BridgeHub, we will need to add some benchmarks for weights caclculation (I guess, I know how, because I was fixing similar stuff for XCMv3 branch)

Reply:
Sure, thank you!